packager/hls/base/simple_hls_notifier.cc

```diff
@@ -41,6 +42,22 @@ bool IsCommonSystemId(const std::vector<uint8_t>& system_id) {
          std::equal(system_id.begin(), system_id.end(), media::kCommonSystemId);
 }
 
+std::string Base64EncodeUri(const std::string& prefix,
+                            const std::string& data) {
+    std::string data_base64;
+    base::Base64Encode(data, &data_base64);
```

Comment:
Not really sure. I went with the assumption that there won't be any weird characters in key_id. I think it could either be 16byte UUID or a numeric id, and it's sent as a POST parameter in player example so there is no need to be urlencoded. Or is there?